pkg/mdata/aggmetrics.go

```diff
@@ -93,7 +93,7 @@ func (ms *AggMetrics) GC() {
 					orgActiveMetrics.Set(float64(len(ms.Metrics[org])))
 					// note: this is racey. if a metric has just become unstale, it may have created a new chunk,
 					// pruning an older one. in which case we double-subtract those points
-					// hard to fix and super rare. see https://github.com/grafana/metrictank/pull/1242
+					// hard to fix and super rare. see https://github.com/grafana/metrictank/pkg/pull/1242
```

Comment:
do you think there's a chance to exclude the patterns `github.com/grafana/metrictank/pull` or `github.com/grafana/metrictank/issue` from your search & replace